source/_posts/2018-01-12-pager-duty-and-confluence.markdown

```diff
@@ -120,13 +116,12 @@ class ConfluenceClient < GrootHTTPClient
     headers = @headers
     headers['Accept'] = 'application/json'
     headers['Content-Type'] = 'application/json'
-    r = RestClient.put(uri, params.to_json, headers)
+    RestClient.put(uri, params.to_json, headers)
   end
 end
 ```
 
-Now let's build a quick and dirty client for PagerDuty:
-
+Now lets build a quick and dirty client for PagerDuty:
```

Comment:
I think your original `let's` (contraction for `let us`) was actually correct :confused: